### PR TITLE
tests(increase-coverage): weekly tests PR

### PR DESCRIPTION
**Pull request by IbrahimCSAE**
### Context

This is a weekly tests PR.

### CODE REVIEW

**Comment by IbrahimCSAE** [comment]
tests failing in the CI are unrelated to my changes

File: tests/Probe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { visitStudy, checkForScreenshot, screenShotPaths, simulateClicksOnElement } from './utils';
+
+test.beforeEach(async ({ page }) => {
+  const studyInstanceUID = '1.3.6.1.4.1.25403.345050719074.3824.20170125095438.5';
+  const mode = 'Basic Viewer';
+  await visitStudy(page, studyInstanceUID, mode);
+});
+
+test('should display the probe tool', async ({ page }) => {
+  await checkForScreenshot(page, page, screenShotPaths.probe.viewportLoaded);
```
Comment (sedghi) [inline tests/Probe.spec.ts:11]: I don't understand why we start by checking the screenshot, that should be a separate test to check loading

**Review by sedghi — COMMENTED** [review]
see my comments please

File: tests/utils/screenShotPaths.ts [hunk 2]
```diff
@@ -2,12 +2,50 @@
  * Paths to the screenshots of the tests.
  */
 const screenShotPaths = {
-  mpr: {
-    mprDisplayedCorrectly: 'mprDisplayedCorrectly.png',
+  angle: {
+    viewportLoaded: 'viewportLoaded.png',
```
[the remark this answers is not in the document]
Comment (IbrahimCSAE) [reply tests/utils/screenShotPaths.ts:6]: The idea is, along side checking that the network is idle, each test should check for an image that indicates the viewer is in a state that’s ready for testing (no black viewports, etc), one reliable way to do it, to get a screenshot of the loaded state, and compare against it until its ready, if that state is not reached in time just fail the test and exit.

i think each test should have it’s own loaded state images, duplicated or not its fine, for example a more advanced test that touches 3D hanging protocols, should have multiple images like 3DOnlyLoaded, 3DPrimaryLoaded, then if you would like to test the trackballrotate tool, you wait first for the viewer state to match that image, then test the tool.

File: tests/Probe.spec.ts (hunk 1 above)
Comment (IbrahimCSAE) [reply tests/Probe.spec.ts:11]: I answered above

File: tests/utils/screenShotPaths.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (IbrahimCSAE) [reply tests/utils/screenShotPaths.ts:6]: I think this is also more reliable than CS3D events cuz they are not always accurate either

**Review by sedghi — APPROVED** [review]

**Pull request merged**